src/fiskaltrust.ifPOS/v1/de/Models/StartExportSessionResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace fiskaltrust.ifPOS.v1.de
+{
+    [DataContract]
+    public class StartExportSessionResponse
+    {
+        [DataMember(Order = 10)]
+        public string TokenId { get; set; }
+
+        [DataMember(Order = 20)]
+        public string TseSerialNumberOctet { get; set; }
```

Comment:
Same as above. Just mentioning this because we should add Base64 to the end if that is used - otherwise, please ignore my comment :)